test/functional/tests/stress/test_stress_without_io.py

```diff
@@ -180,7 +178,12 @@ def test_stress_reload_cache(cache_mode):
                 TestRun.fail(f"Expected cores count: 1; Actual cores count: {cores_count}.")
 
     with TestRun.step("Check md5 of test file."):
-        check_files(core, size_before_load, permissions_before_load, md5_before_load)
+        core.mount(mount_point)
+        file_after = file.get_properties()
+        if file_after != file:
```

Comment:
So here we compare file with it's own properties?
The first is object of `File`, the second is object of `FileProperties`. Doesn't it throw exception or something?

Reply:
No, because object of File (which inherits after FsItem) has the same properties to compare with FileProperties' object. But You're right, for clarity and explicitness there should be compared two objects of the same class. I'll do that.